docs/old-reference-guide/modules/ROOT/pages/known-issues-and-workarounds.adoc

```diff
@@ -0,0 +1,55 @@
+When looking to provide a pull request, be sure to adjust link:https://github.com/AxonFramework/AxonFramework/blob/master/docs/old-reference-guide/modules/ROOT/pages/serialization.adochttps://github.com/AxonFramework/AxonFramework/blob/master/docs/old-reference-guide/modules/ROOT/pages/known-issues-and-workarounds.adoc[this file].
+====
+
+== Relational database storage and serialized data columns
+
+When storing the serialized formats of, for example, xref:axon-framework-reference:events:event-processors/streaming.adoc#tracking-tokens[tokens], xref:axon-framework-reference:sagas:index.adoc[sagas], xref:axon-framework-reference:tuning:event-snapshots.adoc[snapshots], or xref:axon-framework-reference:events:infrastructure.adoc[events], it is good to know Axon uses the (B)LOB for these columns in your RDBMS of choice.
+
+Some databases react unexpectedly to this, with a clear case being link:https://www.postgresql.org/[PostgreSQL].
+It uses a mechanism it calls link:https://wiki.postgresql.org/wiki/TOAST["TOAST"] to move data that's too large to another table.
+Especially (B/C)LOB columns are automatically replaced for the OID type.
+
+Hence, all your tokens, sagas, snapshots, and events will automatically move to another table, with all added overhead coming with it.
+Using xref:axon-server-reference::index.adoc[Axon Server] will, obviously, resolve this for your events and snapshots, as you would no longer use an RDBMS in that case.
+However, even when using Axon Server, the predicament remains for tokens and sagas.
+
+For those cases, it would be wise to replace the TOAST behavior entirely.
+The most straightforward way, is to enforce the use of the `bytea` column type; with some additional steps, of course.
+The following link:https://www.axoniq.io/blog/axonframework-and-postgresql-without-toast[blog post] explains it in great detail, so be sure to check it out.
+
+== Sequence generation issues with JPA Entities
+
+For the JPA entities that require a generated sequence, Axon opted to use the default `@GeneratedValue` annotation back in 2016.
+In the pre-Hibernate-6 age, this typically resulted in the construction of a single sequence generator that would be reused by several tables.
+This can already be regarded as suboptimal, as frequently the sequence generator would be reused between tables.
+
+Especially given the nature of the `global_index` column on the `domain_event_entry` table, which is used to keep the progress in xref:axon-framework-reference:events:event-processors/streaming.adoc[streaming event processors], a reused generator causes unexpected gaps in event processing.
+Although Axon can deal with these gaps, they do make the queries to the `domain_event_entry` table rather bulky.
+
+Just as with any event storage solution, using a dedicate event store implementation like xref:axon-server-reference::index.adoc[Axon Server] will solve the problem entirely.
```

Comment:
Did you mean to say `dedicated` instead of `dedicate`?

Reply:
You are entirely right! Thanks for spotting it. I'll adjust it accordingly.